yarn-project/end-to-end/src/fixtures/cheat_codes.ts

```diff
@@ -256,6 +256,19 @@ export class L2CheatCodes {
     return await this.aztecRpc.getBlockNum();
   }
 
+  /**
+   * Set time of the next execution on L2.
+   * It also modifies time on L1 for next execution and stores this time as the last rollup block on the rollup contract.
+   * @param to - The timestamp to set the next block to (must be greater than current time)
+   * @param rollupContract - The rollup contract address
+   */
+  public async warp(to: number, rollupContract: EthAddress): Promise<void> {
```

Comment:
It should be possible for us to get the `rollupAddress` from the node instead of passing it in.